Resources/Prototypes/_NF/Guidebook/nf14.yml

```
@@ -11,6 +11,7 @@
   - Expeditions
   - CargoHauling # TODO: separate this out along with salvage and mining into some NF_Cargo section
   - Shipyard
+  - WeaponsSystems # mono
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)